test/testutil/mocks/sector_storage/manager_mock.hpp

```diff
@@ -178,8 +178,8 @@ namespace fc::sector_storage {
         const UnpaddedPieceSize &new_piece_size,
         proofs::PieceData piece_data,
         uint64_t priority) override {
-      return doAddPieceSync(
-          sector, piece_sizes, new_piece_size, piece_data.getFd(), priority);
+      int fd = (piece_data.IsNullData()) ? -2 : piece_data.getFd();
```

Comment:
Let's make MOCK function for doAddNullPieceSync.

It will be more clear in the future than magic -2.

Reply:
Fixed